### DISCUSSION TITLE
[Question]: AI PDF plugin is not working?

### DISCUSSION THREAD

**Discussion by githubdebugger** [danny-avila/LibreChat#797] [Q&A] [answered]
### What is your question?

I was trying out plugins on LibreChat, seems like AI PDF plugin is not working, not sure if its just me.

### More Details

 AI PDF errors out with:

Input: {"query":"Summarize the PDF at https://anysamplePDF URL here.pdf"}
Output: 403: Forbidden for upload_and_search_pdf called with ""

However, I tried the same URL with AskYourPDF and that works fine without any issues.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
these plugins are hit or miss with my implementation. mileage can vary based on the default instructions given in the Manifest file or openapi spec. AskYourPDF works better because it has better instructions to that end. I'm moving this to discussions because putting attention on specific plugins will not be a priority. Overall, the chatgpt plugins architecture is not a priority to optimize, it is much more worthwhile for me to spend time on the langchain tooling architecture, and I think these will see increased applications and ecosystem in the long-term.

**Reply by danny-avila** [reply, marked as answer]
For now you can try experimenting with the openapi spec and manifest files yourself. See my guide on how to do this: https://docs.librechat.ai/features/plugins/chatgpt_plugins_openapi.html
